api/policy.go

```diff
@@ -15,11 +17,17 @@ type PipelineApprovalValue string
 // BackupPlanPolicySchedule is value for backup plan policy.
 type BackupPlanPolicySchedule string
 
+// AllowedWindowPeriodCron is value for allowed window period policy, and is stored in cron format.
+// a valid cron expression should be parsed without any error by this: cron.NewParser(cron.Minute | cron.Hour | cron.Dom | cron.Month | cron.Dow)
+type AllowedWindowPeriodCron string
+
 const (
 	// PolicyTypePipelineApproval is the approval policy type.
 	PolicyTypePipelineApproval PolicyType = "bb.policy.pipeline-approval"
 	// PolicyTypeBackupPlan is the backup plan policy type.
 	PolicyTypeBackupPlan PolicyType = "bb.policy.backup-plan"
+	// PolicyTypeAllowedWindowPeriod is the allowed window period type and is stored in cron format
```

Comment:
PolicyTypeAllowedWindow is OK.

Window has the meaning of period.  (e.g. we can just time window instead of time window period)

Reply:
done